.mailmap

```diff
@@ -20,3 +20,4 @@ Renovate Bot[bot]     <bot@renovateapp.com>
 github-actions[bot]   <github-actions@no-reply.github.com>                             <41898282+github-actions[bot]@users.noreply.github.com>
 Carol Barno           <carol.barno.dev@gmail.com>                                      <32536886+CarolBarno@users.noreply.github.com>
 Aaron Cole            <aaron@epiphron.co.nz>
+Jamie Allen           <jamie.allen@metoffice.gov.uk>           JAllen42                <69790602+JAllen42@users.noreply.github.com>
```

Comment:
Hmm, this didn't work. Maybe:
```suggestion
Jamie Allen           <jrallen42@gmail.com>           JAllen42
```

Reply:
I've updated the commit email address, so hopefully that works now